model_prices_and_context_window.json

```diff
@@ -4485,9 +4485,9 @@
         "litellm_provider": "replicate",
         "mode": "chat"
     },
-    "openrouter/deepseek/deepseek-coder": {
```

Comment:
can you not delete the model, instead add a new model 

context - deleting the model has a risk of breaking existing user code